packages/lesswrong/lib/sql/SelectQuery.ts

```diff
@@ -227,14 +229,15 @@ class SelectQuery<T extends DbObject> extends Query<T> {
     table: Table | Query<T>,
     count?: boolean,
     projection?: MongoProjection<T>,
+    joinedFields?: Record<string, 0 | 1 | boolean>,
     autoIncludeId = true,
   ): Atom<T>[] {
     if (count) {
       return ["count(*)"];
     }
 
     if (!projection) {
-      return [this.getStarSelector()];
+      return [[this.getStarSelector(), ...Object.keys(joinedFields || {}).map((field) => `"${field}"`)].join(", ")];
```

Comment:
This is something I'll fix before merging the PR, there is no reason for this to only work if `!projection`